documentation/src/docs/user-guide.template.md

```diff
@@ -1295,6 +1295,55 @@ void restrictedEmailAddresses(@ForAll @Email(quotedLocalPart = false, ipv4Host =
 }
 ```
 
+### Date Arbitrary Types
+
+You can create an arbitrary for date values by calling a static method on class `Dates`:
+
+- [`DateArbitrary dates()`](/docs/${docsVersion}/javadoc/net/jqwik/api/Dates.html#dates())
+- [`YearArbitrary years()`](/docs/${docsVersion}/javadoc/net/jqwik/api/Dates.html#years())
+- [`MonthArbitrary months()`](/docs/${docsVersion}/javadoc/net/jqwik/api/Dates.html#months())
+- [`DaysOfMonthArbitrary daysOfMonth()`](/docs/${docsVersion}/javadoc/net/jqwik/api/Dates.html#daysOfMonth())
+- [`YearMonthArbitrary yearMonths()`](/docs/${docsVersion}/javadoc/net/jqwik/api/Dates.html#yearMonths())
+- [`MonthDayArbitrary monthDays()`](/docs/${docsVersion}/javadoc/net/jqwik/api/Dates.html#monthDays())
+
+In addition, you can constrain their values using the following functions:
+
+#### DateArbitrary
+
```

Comment:
Add a code example or a few for generation of dates. Mention that `LocalDate` is the target type.